File: py/IEC62325/MarketOperations/MarketSystem/ExternalInputs/InterchangeETCData.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106 on Wed Dec 27 15:35:30 2023
from typing import Optional

class InterchangeETCData:
    """
    Existing Transmission Contract data for an interchange schedule.
    @created 27-Dec-2023 3:29:16 PM
    """

    def __init__(self) -> None:
        self.contract_number: Optional[str] = ""  # Existing transmission contract number
        self.usage_mw: float = 0.0  # Existing transmission contract usage MW value

File: java/IEC62325/MarketOperations/MarketSystem/ExternalInputs/InterchangeETCData.java
package IEC62325.MarketOperations.MarketSystem.ExternalInputs;

import IEC61970.Base.Domain.String;
import IEC61970.Base.Domain.Float;

/**
 * Existing Transmission Contract data for an interchange schedule.
 * @created 03-Jan-2024 2:08:10 PM
 */
public class InterchangeETCData {

	/**
	 * Existing transmission contract number
	 */
	public String contractNumber;
	/**
	 * Existing transmission contract usage MW value
	 */
	public Float usageMW;

	public InterchangeETCData(){

	}
}//end InterchangeETCData